content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 100
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and the sidecar injector. By default,
+these webhooks manage their own configurations. From a
+security perspective, this default behavior is not recommended because a compromised webhook could then conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of the webhooks, to
+manage their configurations.
+
+## Before you begin
+
+* Install Istio with [DNS certificates configured](/docs/tasks/security/dns-cert) and
```

Comment:
is this correct?  getting 404 on https://preliminary.istio.io/docs/tasks/security/dns-cert

Reply:
Great catch @linsun The correct path now is: /docs/tasks/traffic-management/ingress/ingress-certmgr/

Reply:
The 404 is because this PR refers to the guide in https://github.com/istio/istio.io/pull/5152, which has not been merged yet. https://github.com/istio/istio.io/pull/5152 should be merged first before merging this PR.

Reply:
Thanks for that context @lei-tang